docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -391,6 +451,88 @@ def my_io_manager(_):
     return MyIOManager()
 ```
 
+### Per-input IO Manager
+
+In some cases you may find that you need to load an input in a way other than the `load_input` function of the corresponding output's IO Manager. For example, let's say Team A has an op that returns an output as a pandas Dataframe and specifies an IO manager that knows how to store and load pandas DataFrames. Your team is interested in using this output for a new op, but you are required to use PySpark to analyze the data. Unfortunately, you don't have permission to modify Team A's IO Manager to support this case. Instead, you can specify an input manager on your op that will override some of the behavior of Team A's IO Manager.
```

Comment:
In the rest of this doc, it looks like we typically write "IO manager" instead of "IO Manager".